lib/hack/WifiUtilities.cpp

```diff
@@ -6,6 +6,7 @@ char wifiTemp[40];
 void printWifiHelp(Print* output) {
   output->println(F("(wi) Wifi info"));
   output->println(F("(ws) Wifi SSID"));
+   output->println(F("(wu) Wifi Username"));
```

Comment:
We should add in the help the (wd) for identify.

I would prefer code in one letter so please use 'd' for identity